azplugins/integrate.py

```diff
@@ -158,3 +158,246 @@ def set_params(self, H=None, V=None, boundary=None):
 
         bc = self._process_boundary(self.boundary)
         self.cpp_method.geometry = _mpcd.SlitGeometry(self.H,self.V,bc)
+
+
+class sinusoidal_channel(_bounce_back):
+    """ NVE integration with bounce-back rules in a anti-symmetric sinusoidal channel.
+
+    Args:
+        group (:py:mod:`hoomd.group`): Group of particles on which to apply this method.
+        A (float): channel cosine amplitude
+        h (float): channel half-width
+        p (int):   channel periodicity
+        boundary : 'slip' or 'no_slip' boundary condition at wall (default: 'no_slip')
+
+    This integration method applies to particles in *group* in the symmetric
+    cosine channel geometry.
```

Comment:
I'm not sure if this is defined elsewhere, but I would appreciate having a mathematical equation here showing how the walls are defined based on the input parameters.